integrations/sbt-bloop/src/sbt-test/sbt-bloop/non-compiling/test

```diff
@@ -0,0 +1,3 @@
+> bloopGenerate
+> test:bloopGenerate
+> checkBloopFiles
```

Comment:
How does this test check that the compilation is not triggered?

Reply:
What we really want to check is that the files are properly generated despite the error. Previously, when we used fullclasspath the `bloopInstall` task itself would fail and nothing would get generated

Reply:
Oh I see there is an error in the `Obj.scala` file